Geometry/EcalCommonData/plugins/dd4hep/DDEcalBarrelNewAlgo.cc

```diff
@@ -0,0 +1,2179 @@
+#include "DD4hep/DetFactoryHelper.h"
+#include "DD4hep/Printout.h"
+// FIXME: use local definition until dd4hep fixes the uniits
+//#include "DataFormats/Math/interface/GeantUnits.h"
+#include "DetectorDescription/DDCMS/interface/DDPlugins.h"
+#include "DetectorDescription/DDCMS/interface/BenchmarkGrd.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "Geometry/CaloGeometry/interface/EcalTrapezoidParameters.h"
+#include "Math/AxisAngle.h"
+#include "CLHEP/Geometry/Point3D.h"
+#include "CLHEP/Geometry/Vector3D.h"
+#include "CLHEP/Geometry/Transform3D.h"
+
+using namespace std;
+using namespace cms;
+using namespace dd4hep;
+
+// FIXME: use local definition until dd4hep fixes the uniits
+// using namespace geant_units::operators;
+
```

Comment:
@ianna @slava77 @perrotta documenting this for people not attending the discussion at the simulation meeting https://indico.cern.ch/event/832351/contributions/3487361/attachments/1875459/3088175/DD4hep_Ecal_Migration.pdf
https://indico.cern.ch/event/832351/contributions/3487361/attachments/1875459/3088218/DD4hep_Migration_Summary_05.07.2019.pdf : ideally here one should just use DataFormats/Math, but this is temporarily disabled because of a mismatch of basic units between Geant4 (1 is 1 mm) and DD4hep (1 is 1 cm). This is going to be fixed in a future patch to ROOT (internally used by DD4hep).